src/test/scripts/installDependencies.R

```diff
@@ -21,22 +21,32 @@
 #
 #-------------------------------------------------------------
 
+print("Starting install RScripts")
+
+args <- commandArgs(TRUE)
+
 custom_install <- function(pkg) {
     if(!is.element(pkg, installed.packages()[,1])) {
-		# https://cran.r-project.org/mirrors.html
-		# Using Wirtschaftsuniversität Wien Mirror
- 		install.packages(pkg, repos="https://cran.wu.ac.at/");
+		# Installing to temp folder, if you want to permenently install change lib path
+		if (length(args)==0) {
+ 			install.packages(pkg);
+		} else if (length(args) == 1){
+			install.packages(pkg, lib= args[1]);
+		}
 	}
 } 
 
 custom_install("Matrix");
 custom_install("plotrix");
```

Comment:
I get the following when executing the script: 
```
Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
Warning message:
package ‘plotrix’ is not available (for R version 3.4.4) 
```

Reply:
true, i should remove this import.